docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2138,8 +2138,8 @@ None
 #### Results:
 
 - `<object>`
-  - `full_snapshot_slot: <u64>` - Highest full snapshot slot
-  - `incremental_snapshot_slot: <u64 | null>` - Highest incremental snapshot slot _based on_ `full_snapshot_slot`
+  - `full: <u64>` - Highest full snapshot slot
+  - `incremental: <u64 | null>` - Highest incremental snapshot slot _based on_ `full`
```

Comment:
I think `null` might actually be `undefined` here (serde_json serialization of `None`)?

Reply:
Done!